modules/elasticsearch/src/test/java/org/testcontainers/elasticsearch/ElasticsearchContainerTest.java

```diff
@@ -0,0 +1,103 @@
+package org.testcontainers.elasticsearch;
+
+
+import org.apache.http.auth.AuthScope;
+import org.apache.http.auth.UsernamePasswordCredentials;
+import org.apache.http.client.CredentialsProvider;
+import org.apache.http.impl.client.BasicCredentialsProvider;
+import org.apache.http.util.EntityUtils;
+import org.elasticsearch.client.Request;
+import org.elasticsearch.client.Response;
+import org.elasticsearch.client.ResponseException;
+import org.elasticsearch.client.RestClient;
+import org.junit.After;
+import org.junit.Test;
+
+import java.io.IOException;
+
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.rnorth.visibleassertions.VisibleAssertions.assertThrows;
+
+public class ElasticsearchContainerTest {
+
+    /**
+     * Elasticsearch default username, when secured with a license > basic
+     */
+    private static final String ELASTICSEARCH_USERNAME = "elastic";
+
+    /**
+     * Elasticsearch 5.x default password. In 6.x images, there's no security by default as shipped with a basic license.
+     */
+    private static final String ELASTICSEARCH_PASSWORD = "changeme";
+
+    private ElasticsearchContainer container = null;
+    private RestClient client = null;
+
+    @After
+    public void stopRestClient() throws IOException {
+        if (client != null) {
+            client.close();
+            client = null;
+        }
+    }
+
+    @After
+    public void stopContainer() {
+        if (container != null) {
+            container.stop();
+        }
+    }
+
+    @Test
+    public void elasticsearchDefaultTest() throws IOException {
+        container = new ElasticsearchContainer();
```

Comment:
We can use the try-with-resources pattern here, which means we wouldn't need the `stopContainer` method (we use it for other tests). No big deal, though.